src/linter-report/simple.js

```diff
@@ -0,0 +1,28 @@
+'use babel'
+
+import { generateRange } from 'atom-linter'
+
+/**
+ * Turn the given options into a Linter message array
+ * @param  {TextEditor} textEditor The TextEditor to use to build the message
+ * @param  {Object} options    The parameters used to fill in the message
+ * @param  {string} [options.severity='error'] Can be one of: 'error', 'warning', 'info'
+ * @param  {string} [options.excerpt=''] Short text to use in the message
+ * @param  {string|Function} [options.description] Used to provide additional information
+ * @return {Array}            Message to user generated from the parameters
+ */
+const simpleReport = (textEditor, {
+  severity = 'error',
+  excerpt = '',
+  description,
+} = {}) => [{
+  severity,
+  excerpt,
+  description,
+  location: {
+    file: textEditor.getPath(),
```

Comment:
📝 Why aren't we just using the path from ESLint? Just a note here, I realize the old code was already doing it this way.

Reply:
I don't know. Sounds like a good suggestion to me.